Frontend/tailored-brand-poc/src/app/all-invoices/all-invoices.component.html

```diff
@@ -76,7 +76,7 @@
         </thead>
         <tbody>
           <tr *ngFor="let data of invoiceData; let i = index">
-            <td>{{ i+1 }}</td>
+            <td>{{ rowId + i }}</td>
```

Comment:
why is width `15%` ?

Reply:
If the width isn't 15% , the input gets cramped